docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+- **Description:** The token that is used for identity authentication within the StarRocks cluster to which the FE belongs. If this parameter is left unspecified, StarRocks generates a random token for the cluster at the time when the leader FE of the cluster is started for the first time.
+
+##### tmp_dir
+
+- **Default:** StarRocksFE.STARROCKS_HOME_DIR + "/temp_dir"
+- **Description:** The directory that stores temporary files such as files generated during backup and restore procedures. After these procedures finish, the generated temporary files are deleted.
+
+##### locale
+
+- **Default:** zh_CN.UTF-8
+- **Description:** The character set that is used by the FE.
+
+##### hive_meta_load_concurrency
+
+- **Default:** 4
+- **Description:** The maximum number of concurrent threads that are supported for Hive metadata.
+
+##### hive_meta_cache_refresh_interval_s
+
+- **Default:** 7200
+- **Description:** The time interval at which the cached metadata of Hive external tables is updated. Unit: second.
+
+##### hive_meta_cache_ttl_s
+
+- **Default:** 86400
+- **Description:** The amount of time after which the cached metadata of Hive external tables expires. Unit: second.
+
+##### hive_meta_store_timeout_s
+
+- **Default:** 10
+- **Description:** The amount of time after which a connection to a Hive metastore times out. Unit: second.
+
+##### es_state_sync_interval_second
+
+- **Default:** 10
+- **Description:** The time interval at which the FE obtains Elasticsearch indexes and synchronizes the metadata of StarRocks external tables. Unit: second.
+
+##### enable_auth_check
+
+- **Default:** TRUE
+- **Description:** Specifies whether to enable the authentication check feature. Valid values: `TRUE` and `FALSE`. `TRUE` specifies to enable this feature, and `FALSE` specifies to disable this feature.
+
+##### enable_metric_calculator
+
+- **Default:** TRUE
+- **Description:** Specifies whether to enable the feature that is used to periodically collect metrics. Valid values: `TRUE` and `FALSE`. `TRUE` specifies to enable this feature, and `FALSE` specifies to disable this feature.
+
+## BE configuration items
+
+Some BE configuration items are dynamic parameters which you can set them by commands when BE nodes are still online. The rest of them are static parameters. You can only set the static parameters of a BE node by changing them in the corresponding configuration file **be.conf**, and restart the BE node to allow the change to take effect.
+
+### View BE configuration items
+
+You can view the BE configuration items using the following command:
+
+```shell
+curl http://<BE_IP>:<BE_HTTP_PORT>/varz
+```
+
+### Configure BE dynamic parameters
+
+You can configure a dynamic parameter of a BE node by using the `curl` command.
+
+```Shell
+curl -XPOST http://be_host:http_port/api/update_config?<configuration_item>=<value>
+```
+
+BE dynamic parameters are as follows.
+
+#### report_task_interval_seconds
+
+- **Default:** 10 seconds
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/fb678dd5-808b-4402-8b7e-82cd74acf19d)
Change "**Default:** 10 seconds" to "**Default:** 10", and add "Unit: seconds" preceding "Default", I think.